.github/workflows/tests.yml

```diff
@@ -59,9 +59,15 @@ jobs:
           restore-keys: tests-php-${{ matrix.php }}-composer
 
       - name: Install Composer dependencies
-        env:
-          PHP_VERSION: ${{ matrix.php }}
-        run: composer install --ansi --no-progress --prefer-dist $(if [ "$PHP_VERSION" == "8.0" ]; then echo "--ignore-platform-reqs"; fi;)
+        run: composer install --ansi --no-progress --prefer-dist ${{ matrix.php == '8.0' && '--ignore-platform-reqs' || '' }}
 
       - name: Run PHPUnit tests
         run: vendor/bin/phpunit
+
+      - name: Send coverage to Coveralls
```

Comment:
What's this service?

Reply:
Coveralls shows historical and per file coverage.
https://github.com/predis/predis/actions/runs/3972188215/jobs/6809833063

Reply:
Please see what a beauty she is https://coveralls.io/jobs/113336063
You can open the directory tree and see where predis may improve.

Reply:
Can we get the source code available?

https://coveralls.io/jobs/113336063/source_files/2021808842

Reply:
Yes. Please add this repo to your Coveralls account.
https://github.com/predis/predis/pull/1036?notification_referrer_id=NT_kwDOAA6Gx7E1Mzg1OTg5NzYyOjk1MjAwNw#issue-1551602274

Reply:
We could have a badge too.
```
[![Coverage Status](https://coveralls.io/repos/github/predis/predis/badge.svg?branch=main)](https://coveralls.io/github/predis/predis?branch=main)
```
[![Coverage Status](https://coveralls.io/repos/github/predis/predis/badge.svg?branch=main)](https://coveralls.io/github/predis/predis?branch=main)